packages/editor/src/components/entities-saved-states/entity-record-item.js

```diff
@@ -64,11 +56,6 @@ export default function EntityRecordItem( { record, checked, onChange } ) {
 					className="entities-saved-states__change-control"
 				/>
 			</PanelRow>
-			{ hasPostMetaChanges && (
```

Comment:
This is the only difference that I made (the "post meta") indicator is removed. For me, this was way too ad-hoc and doesn't really solve any particular issue to be worth keeping.